Source/Core/Core/HW/EXI/BBA/TAP_Apple.cpp

```diff
@@ -48,12 +48,13 @@ bool CEXIETHERNET::TAPNetworkInterface::IsActivated()
 
 bool CEXIETHERNET::TAPNetworkInterface::SendFrame(const u8* frame, u32 size)
 {
-  INFO_LOG(SP1, "SendFrame %x\n%s", size, ArrayToString(frame, size, 0x10).c_str());
+  INFO_LOG_FMT(SP1, "SendFrame {}\n{}", size, ArrayToString(frame, size, 0x10));
```

Comment:
I'm fine with the size being printed as a decimal number rather than hex without prefix. It's consistent with the `TAP_Win32` wording but not `TAP_Unix`.